src/girouette/tw/border.cljc

```diff
@@ -4,17 +4,18 @@
 (def components
   [{:id     :border
     :rules  "
-    border = <'border'> (<'-'> direction)? (<'-'> integer)?
+    border = <'border'> (<'-'> direction)? (<'-'> border-width)?
+    border-width = number | length | length-unit
```

Comment:
For the `:id` and main rule of the component's grammar, I usually choose the title of the section in Tailwind's documentation.

Here, that would be something like:
```clojure
{:id :border-width
 :rules "
 border-width = <'border'> (<'-'> direction)? (<'-'> border-width-value)?
 border-width-value = number | length | length-unit
"
```